resources/core/charts/logging/README.md

```diff
@@ -0,0 +1,49 @@
+
+```
+  _                       _             
+ | |                     (_)            
+ | |     ___   __ _  __ _ _ _ __   __ _ 
+ | |    / _ \ / _` |/ _` | | '_ \ / _` |
+ | |___| (_) | (_| | (_| | | | | | (_| |
+ |______\___/ \__, |\__, |_|_| |_|\__, |
+               __/ | __/ |         __/ |
+              |___/ |___/         |___/ 
+```
+
+## Overview
+This document explains how Kyma installs `OK Log` and `Logspout` in the `kyma-system` namespace, and how to use it to check logs in Kyma.
+
+## Details
+The chart installs the following resources in `kyma-system` namespace:
```

Comment:
Please capitalize all instances of Namespace.

Reply:
done